test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-snapshotter.yaml

```diff
@@ -0,0 +1,49 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-hostpath-snapshotter
+  labels:
+    app: csi-hostpath-snapshotter
+spec:
+  selector:
+    app: csi-hostpath-snapshotter
+  ports:
+  - name: dummy
+    port: 12345
+
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-hostpath-snapshotter
+spec:
+  serviceName: "csi-hostpath-snapshotter"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-hostpath-snapshotter
+  template:
+    metadata:
+      labels:
+        app: csi-hostpath-snapshotter
+    spec:
+      serviceAccount: csi-snapshotter
+      containers:
+      - name: csi-snapshotter
+        image: quay.io/k8scsi/csi-snapshotter:v0.4.1
+        args:
+        - "--snapshotter=csi-hostpath"
+        - "--csi-address=$(ADDRESS)"

```

Comment:
we should clean this up in the csi examples too, but there's not really a need to define an environment variable for a value that is hardcoded.

Reply:
Got it.